timemachine/fe/bar.py

```diff
@@ -132,13 +132,16 @@ def bootstrap_bar(w_F, w_R, n_bootstrap=1000, timeout=10):
 
     t0 = time()
 
+    seed = 2022
+    rng = np.random.RandomState(seed)
+
     for _ in range(n_bootstrap):
         elapsed_time = time() - t0
         if elapsed_time > timeout:
             break
 
-        inds_F = np.random.randint(0, n_F, n_F)
-        inds_R = np.random.randint(0, n_R, n_R)
+        inds_F = rng.randint(0, n_F, n_F)
+        inds_R = rng.randint(0, n_R, n_R)
```

Comment:
nit: to me it's slightly more obvious to do something like
```suggestion
        w_F_sample = rng.choice(w_F, replace=True)
        w_R_sample = rng.choice(w_R, replace=True)
```
(e.g. don't need to think about whether the bounds of `randint` are inclusive or not)

Reply:
Applied. One reason to prefer `inds` in the future might be if we model correlation between neighboring pair bar estimates (0 -> 1, 1 -> 2 are not independent, and could reference the same bootstrap sample from lambda idx 1 twice), but this is not possible with the current signature anyway.

Reply:
Makes sense. In that case I think you could still use `inds_F = rng.choice(n_F, size=(n_F,))` and not have to think about inclusivity of bounds.